src/packages/client/src/runtime/utils/find.ts

```diff
@@ -0,0 +1,290 @@
+/* eslint-disable @typescript-eslint/no-inferrable-types */
+import fs from 'fs'
+import path from 'path'
+
+const readdirAsync = fs.promises.readdir
+const realpathAsync = fs.promises.realpath
+const statAsync = fs.promises.stat
```

Comment:
We didn't use `fs.promises` until now because on Node 10 it's marked as experimental and prints warnings in logs.

We can still merge or do it for next release and drop Node 10 at the same time.

Reply:
Ok, will add `promisify` instead